docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -34,6 +34,19 @@ If the application resumes data successfully:
     * `SetGlobalProperties`
     * `SubscribeButton`
     * `SubscribeVehicleData`
+    * `SubscribeWayPoints`
+    * `CreateWindow`
+    * `GetInteriorVehicleData`
+
+If HMI responds with any kind of error or does not respond to any requests sent during resumption, SDL must revert already restored data with appropriate RPCs:
+    * `DeleteCommand`
+    * `DeleteSubMenu`
+    * `DeleteInteractionChoiceSet`
+    * `ResetGlobalProperties`
+    * `UnsubscribeButton`
+    * `UnsubscribeVehicleData`
+    * `UnsubscribeWayPoints`
+    * `DeleteWindow`
```

Comment:
please add GetInteriorVehicleData(moduleType, moduleID, subscribe=false)

Reply:
thank you, done in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/225/commits/9074c7d61dea3f16ee213393369743b367f7b50d